mediajson/decode.py

```diff
@@ -87,6 +87,16 @@ def decode_value(o: JSONSerialisable) -> MediaJSONSerialisable:
 
 
 class NMOSJSONDecoder(JSONDecoder):
+    def __init__(self, **kwargs):
+        # Filter out the 'encoding' parameter as a simple workaround for simplejson adding it.
+        # The parameter is no longer supported in python 3.
+        py3_kwargs = {
+            key: value
```

Comment:
This is beautiful! It looks like a haiku. :chef's kiss:.

Reply:
Structure was copied from code @jamesba wrote :smile: